index.js

```diff
@@ -106,12 +110,35 @@ function SonicBoom (opts) {
   this.maxLength = maxLength || 0
   this.maxWrite = maxWrite || MAX_WRITE
   this.sync = sync || false
+  this.writable = true
```

Comment:
forgot to mention, this is so that node.js stream util compose function works properly with this